curve_tracer_controller:
  theta_precision: 0.5
  distance_precision: 0.1
  pid: {p: 0.5, i: 0, d: 0}